docs/usage/installation.md

```diff
@@ -11,14 +11,19 @@ Other installation locations are also possible, but please keep the original lay
 
 ## From Godot's Asset Library
 
-_To be written..._
+_Godot-Python is in early development and is not yet available on the Godot Asset Library._
 
 ## From GitHub releases
 
-Visit the project's [releases page](https://github.com/maiself/godot-python-extension/releases) and download the archive for the version of the extension you need. Then extract the archive into the `addons/godot-python-extension` directory within your project.
+_Godot-Python is in early development and does not yet have an official release._
 
+[//]: # (Visit the project's [releases page]&#40;https://github.com/maiself/godot-python-extension/releases&#41; and download the archive for the version of the extension you need. Then extract the archive into the `addons/godot-python-extension` directory within your project.)
 
-## As a submodule
+## From nightly builds (GitHub Actions)
+
+You can download nightly builds from [`GitHub Actions`](https://github.com/maiself/godot-python-extension/actions). Keep in mind nightly builds may be unstable and are not recommended for production setups.
+
+## As a git submodule
```

Comment:
Changing the sub heading name broke cross referencing below (`#as-a-submodule`), change one or the other so they match again.